example-secrets/docker-build-secrets.yml.example

```diff
@@ -0,0 +1,6 @@
+docker-registry-ci: eu.gcr.io/census-rm-ci
+docker-registry-gcr: eu.gcr.io/census-gcr
+gcp-service-account-json-ci:
+gcp-service-account-json-gcr:
```

Comment:
I don't think we should be using service accounts like this here, we have a concourse SA you can use with `((gcp.service_account_json))`

Reply:
Fixed.